metadata/updater/updater.go

```diff
@@ -234,10 +234,11 @@ func (update *Updater) DownloadTarget(targetFile *metadata.TargetFiles, filePath
 			hashes = hex.EncodeToString(v)
 			break
 		}
-		dirName, baseName, ok := strings.Cut(targetFilePath, "/")
+		baseName := filepath.Base(targetFilePath)
+		dirName, ok := strings.CutSuffix(targetFilePath, "/"+baseName)
```

Comment:
I know it's just a minor difference in this case but when it comes to file paths I always prefer things like `filepath.Join` instead of direct string concatenation. It's also more OS independent usually.

Reply:
I completely agree! Especially for windows builds. However, in this case the `targetFilePath` comes from metadata and I believe that this is standardized to use `/` only and if you use `filepath.Join()` on windows you end up generating an incorrect `targetRemotePath`. 

This was recently fixed in https://github.com/kommendorkapten/go-tuf/commit/484e95dc84e3daf7ee6a0933e159a257126670d0 to always use `/` for remote paths.

Reply:
Ah I see, then we should be good here.